app/services/notification_service.py

```
@@ -0,0 +1,534 @@
+            data=data,
+            recipient=recipient_override or await self._get_recipient(user, notification_type)
+        )
+        
+        db.add(notification)
+        await db.commit()
+        await db.refresh(notification)
+        
+        # Send notification asynchronously
+        asyncio.create_task(
+            self._deliver_notification(db, notification)
+        )
```

Ruff (0.13.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)